Competition2017/src/competition/subsystems/climbing/ClimbingSubsystem.java

```diff
@@ -13,16 +13,18 @@
 
 @Singleton
 public class ClimbingSubsystem extends BaseSubsystem {
+
     private static Logger log = Logger.getLogger(ClimbingSubsystem.class);
 
-    private final DoubleProperty ascendPowerProperty;
-    private final DoubleProperty descendPowerProperty;
-    private final XCANTalon climbingMotor;
-    
+    protected final DoubleProperty ascendPowerProperty;
+    protected final DoubleProperty descendPowerProperty;
+    public final XCANTalon climbingMotor;
```

Comment:
If you're using a test wrapper class (TestClimbingSubsystem), then you should go ahead and make the climbingMotor protected instead of public, and then put a getter on the TestClimbingSubsystem to get it.